src/control/Control.h

```diff
@@ -385,6 +388,11 @@ class Control:
     int autosaveTimeout = 0;
     fs::path lastAutosaveFilename;
 
+    /**
+     * The publish handler ID
+     */
+    int publishTimeout = 0;
```

Comment:
This name is confusing; I initially thought this was the publication interval. Can you rename this to something like `autoPublishHandler`?